packages/jest-circus/src/legacy-code-todo-rewrite/jestAdapterInit.ts

```diff
@@ -5,6 +5,7 @@
  * LICENSE file in the root directory of this source tree.
  */
 
+import type BabelTraverse from '@babel/traverse';
```

Comment:
is this in `package.json`?

Reply:
[Yup](https://github.com/facebook/jest/blob/master/packages/jest-circus/package.json#L13) :)